internal/app/feed/exporter_csv.go

```diff
@@ -74,6 +74,18 @@ func (e *CSVExporter) FinaliseExport(feed Feed, rows interface{}) error {
 		}
 
 		if file == nil || rowsAdded >= e.MaxRowsPerFile {
```

Comment:
This switching logic has a bit of a code smell.

I feel like this is all quite error prone especially if someone comes in later and makes a change.

Would you be able to refactor this? Potentially something like:
1. If over row limit -> Close and rename file
2. Check if file exists, if not create it.